bolt-sidecar/src/builder/state_root.rs

```diff
@@ -15,11 +15,15 @@ mod tests {
     #[tokio::test]
     async fn test_trace_call() -> eyre::Result<()> {
         dotenvy::dotenv().ok();
-        tracing_subscriber::fmt::init();
+        let _ = tracing_subscriber::fmt::try_init();
+
+        let Some(rpc_url) = std::env::var("RPC_URL").ok() else {
+            tracing::warn!("RPC_URL not found in environment variables, skipping test");
+            return Ok(());
+        };
```

Comment:
we have `test_util::try_get_execution_api_url()` which will try to ping the EL from our Helder tailnet and use it.
Env vars are also a great option, just wanted to point this out